src/ensemble/basic_ensemble_solve.jl

```diff
@@ -109,7 +122,7 @@ function batch_func(i, prob, alg; kwargs...)
         iter += 1
         _prob = prob.safetycopy ? deepcopy(prob.prob) : prob.prob
         new_prob = prob.prob_func(_prob, i, iter)
-        x = prob.output_func(solve(new_prob, alg; kwargs...), i)
+        x = prob.output_func(solve(new_prob, alg; progress_name, progress_id, kwargs...), i)
```

Comment:
Are you able to setup a downstream weakdep so that this is forced to be on a version where it will work?

Reply:
I don't think I can set a version constraint that works until there is a version with https://github.com/SciML/DiffEqBase.jl/pull/939 so that should be merged and released first so I can then constrain this to require that version.